### ISSUE TITLE
Bulk Enquiries

### ISSUE THREAD

**Issue by pol76** [AyeCode/geodirectory#777]
Many customers asked this in the past and I can see why from GeoDirectory Experts.

It would be great if a user could send the same message to multiple listings without having to copy and paste the message on each listing contact form.

There could be a checkbox in each listing contact form asking the user if he wants to send the same message to other listings. If checked few options should be displayed like:

Send to all listings in the directory
Send to all listings in the category
Send to all listings in the location
Send to all listings in the category + location

Let me know how feasible this is and what you think.

Thanks!

**Comment by Guust-Flater** [comment]
Send to all listings in the directory
Send to all listings in the category
Send to all listings in the location
Send to all listings in the category + location

That sounds like a recipe for spamming. That may result in listing owners removing their contact details or listing. 

What about some integration with GD lists? Build your own private list, and then send a query to that list?
Site owners can limit the numbers that can be added to such a list to minimize abuse.

None of the options suggested by  Paolo allow for a targeted question for example if I want to check availability for some selected hotels in a large area with lots of listings, but I am not interested in the majority of them. 

Just my thoughts, as always.

**Comment by pol76** [comment]
For a generic directory, I'm sure it could be used to spam, but we could make optional and assist directories were this is done daily like on gd experts. I see the same email sent to all experts almost every day. 
Something with lists could be another option.

**Comment by Stiofan** [comment]
I agree we need something, finding the right way to do it is the most important part.

Giving a user the option to send to many could lead to a higher contact form rate for owners but less conversions for them.

**Comment by Stiofan** [comment]
Similar request by user: https://wpgeodirectory.com/support/topic/membership-options-question/
I think integrating this with fav's and lists might be the best route?

**Comment by alexrollin** [comment]
Similar request - with some ideas about a query to select listing recipients

https://secure.helpscout.net/conversation/1586398232/22616/

**Comment by Stiofan** [comment]
Another https://wordpress.org/support/topic/multi-enquiry-listings/
